addons/project_todo/static/src/components/todo_chatter/todo_chatter.js

```diff
@@ -0,0 +1,21 @@
+import { onPatched } from "@odoo/owl";
+import { scrollTo } from "@web/core/utils/scrolling";
+import { Chatter } from '@mail/chatter/web_portal/chatter';
+
+export class TodoChatter extends Chatter {
+    setup() {
+        super.setup();
+        this.isChatterScroll = false;
+        onPatched(() => {
+            if (this.isChatterScroll) {
+                this.isChatterScroll = false;
+                scrollTo(document.querySelector('.o_FormRenderer_chatterContainer'), { behavior: "smooth" })
+            }
+        });
+    }
+
+    _onMounted() {
+        super._onMounted();
+        this.isChatterScroll = true;
+    }
+}
```

Comment:
Actually, what is the purpose of these changes?

Reply:
Ok I see, then I think we could do it in TodoPanel instead of creating an override just for that. I will do it.

Reply:
Okay, I checked the code and noticed that the chatter always displays on small devices. However, the testing point was to enable scrolling down in the small view, so we had to extend the chatter. thanks